apps/src/studio/studio.js

```diff
@@ -3013,7 +3021,14 @@ Studio.displaySprite = function(i, isWalking) {
     spriteWalkIcon.setAttribute('visibility', 'hidden');
 
     xOffset = sprite.drawWidth * spriteFrameNumber(i);
-    yOffset = 0;
+
+    // For new versions of sprites (iceage/gumball), there are additional rows of
```

Comment:
:)

Agree that having comments explaining which skins this affects is probably fine.